spec/index.bs

```diff
@@ -15516,6 +15516,11 @@ Rasterization produces a list of <dfn dfn>RasterizationPoint</dfn>s, each contai
         |rp|.[=RasterizationPoint/destination=].[=FragmentDestination/position=]
         outside of |state|.{{RenderState/[[scissorRect]]}}.
 
+    1. If the `frag_depth` [=builtin=] is not produced by the fragment shader remove all the points
+        |rp| from |rasterizationPoints| that fail the
+        {{GPURenderPipelineDescriptor/depthStencil}}.{{GPUDepthStencilState/depthCompare}}
+        operation.
```

Comment:
Realized that handling early depth fail like this won't work because we may still need to process the failed rasterization points in depth/stencil processing for the stencil's `depthFailOp`. Also need to ensure that we're able to handle backface stencil ops correctly. :weary: